libcxx/include/__bit/countl.h

```diff
@@ -38,6 +41,9 @@ _LIBCPP_NODISCARD inline _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR int __libcpp_cl
 
 #ifndef _LIBCPP_HAS_NO_INT128
 inline _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR int __libcpp_clz(__uint128_t __x) _NOEXCEPT {
+#  if __has_builtin(__builtin_clzg)
```

Comment:
These headers seem to not put whitespace after the `#`. Mind staying consistent in this file (and libcxx/include/__bit/popcount.h)? Then we should be good to go.  Perhaps your editor is trying to auto indent these.

Reply:
No, the indentation is correct.

Reply:
No. This is a clang-format thing. The indentation happens because we have nested #if's here.

Reply:
ah, ok LGTM then. Thanks for your work on this patch!